wizard/.streamlit/config.toml

```diff
@@ -0,0 +1,9 @@
+[browser]
+gatherUsageStats = false
+
+[theme]
+primaryColor="#F29828"
```

Comment:
Space around ` = ` and missing carriage return at the EOF.

Can you maybe install a formatter for toml on your IDE?